UnityProject/Assets/Scripts/Managers/NetworkManagement/SpawnHandler.cs

```diff
@@ -58,6 +58,13 @@ public static void TransferPlayer(NetworkConnection conn, short playerController
 		{
 			playerScript.PlayerSync.NotifyPlayers(true);
 		}
+
+		var playerObjectBehavior = newBody.GetComponent<ObjectBehaviour>();
+		if (playerObjectBehavior.parentContainer)
+		{
+			ClosetHandlerMessage.Send(newBody, playerObjectBehavior.parentContainer.gameObject);
```

Comment:
I think some comments explaining why this is being done would be very helpful to anyone looking at this in the future.